weak-modules2

```diff
@@ -380,15 +382,8 @@ run_depmod_and_mkinitrd() {
 		mkdir -p /run/regenerate-initrd
 		touch /run/regenerate-initrd/$image-$krel
 	    else
-		doit /sbin/mkinitrd -k /boot/$image-$krel -i /boot/initrd-$krel
+		doit "$DRACUT" -f /boot/initrd-$krel
```

Comment:
This is wrong, too.

Reply:
Solved in 16.0.2 (89030b5, fc1c54a)

Reply:
you can also call `dracut -f --kver $krel`

Reply:
I know. I decided to use the more conventional syntax. It really doesn't matter. Only **not** specifying the kernel version at all, as I did in this PR, was wrong.

Reply:
Luckily the broken code hadn't been submitted to Factory yet.